cek.go

```diff
@@ -0,0 +1,29 @@
+package mssql
+
+type cekTable struct {
```

Comment:
what is cek mean? A comment or full name would be good.

Reply:
Column Encryption Key, this is used everywhere in Microsoft's documentation. Not sure if it's a good idea to make it longer here since `cek` is mentioned everywhere in the codebase now

Reply:
Okay, then above `type cekTable struct {`, put
```
// Column Encryption Key table that ...
```